packages/desktop-client/src/components/payees/index.js

```diff
@@ -544,7 +548,7 @@ export const ManagePayees = forwardRef(
           <View
             style={{
               flex: 1,
-              border: '1px solid ' + colors.border,
+              border: '1px solid ' + theme.tableBorder,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏the payee table has quite a few changes too

- [ ] "create rule" button background color is wrong
- [ ] table is missing row `:hover` effect
- [ ] table content color is different
- [ ] status texts have different color (i.e. search for some payee that does not exist to see "no payee" text)
- [ ] selected payee text color is wrong (screenshot below)


before:
<img width="294" alt="Screenshot 2023-08-12 at 09 48 51" src="https://github.com/actualbudget/actual/assets/886567/7311fb3b-338e-4039-8dff-8ada7aa95d1a">


after:
<img width="317" alt="Screenshot 2023-08-12 at 09 48 22" src="https://github.com/actualbudget/actual/assets/886567/502c3b64-6c3f-4b11-b0a9-ec73e3fc72ce">

Reply:
All fixed.